### ISSUE TITLE
des MEDS writing row,col convention mismatch

### ISSUE THREAD

**Issue by esheldon** [GalSim-developers/GalSim#727]
The convention in MEDS is that row,col corresponds to x,y in the image in the following way
```
row -> y
col -> x
```
At these lines they are swapped

https://github.com/GalSim-developers/GalSim/blob/master/galsim/des/des_meds.py#L258

Those should read
```python
            dudrow[i] = obj.wcs[i].dudy
            dudcol[i] = obj.wcs[i].dudx
            dvdrow[i] = obj.wcs[i].dvdy
            dvdcol[i] = obj.wcs[i].dvdx
            row0[i]   = obj.wcs[i].origin.y
            col0[i]   = obj.wcs[i].origin.x
```

**Comment by rmandelb** [comment]
There is already an open PR for updates to the DES code ( https://github.com/GalSim-developers/GalSim/pull/706 ), so perhaps this could be done there?
(+unit test that would catch the swap?)

**Comment by rmjarvis** [comment]
Yep.  Thanks Erin!  I'm embarrassed that I didn't notice this myself, since I've been getting on your case to get this right in the ngmix code.  :)  I fixed it here: 5bad7ce (on branch #654)
